### ISSUE TITLE
Accessibility: cannot close popup dialogs with the screen reader

### ISSUE THREAD

**Issue by vick08** [google-gemini/gemini-cli#12857]
### What happened?

When a dialog with a "close" button pops up, e.g. when Gemini-cli cannot reach an MCP server, there seems to be no way to idenitfy the "close" button and activate it in order to close the dialog.

### What did you expect to happen?

I was expecting to see a clear identification of the "close" button with my screen reader as well as a keyboard shortcut or another command to close the dialog.

### Client information

<details>
<summary>Client Information</summary>

Run `gemini` to enter the interactive CLI, then run the `/about` command.

```console
> /about
# paste output here
```

About Gemini CLI
CLI Version 0.13.0
Git Commit 230056cc
Model auto
Sandbox no sandbox
OS darwin
Auth Method gemini-api-key
</details>

### Login information

API key
